lib/routes/admin-api/tag-schema.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const joi = require('joi');
+const { customJoi } = require('./util');
+
+const tagSchema = joi
+    .object()
+    .keys({
+        value: joi.string().min(2),
```

Comment:
does it make sense to enforce a max length?

Reply:
Maybe, we definitely do not want 100s of chars in this string; but what would be a decent length where you still have room for your team name if you're Team-External-Sales, I guess you'd be upset if we only allowed 10, so maybe 30 or 40?